### PR TITLE
Prepare minor release

### PR DESCRIPTION
**Pull request by bpoldrack**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -36,9 +32,8 @@ bet we will fix some bugs and make a world even a better place.
   without decoding it if the line didn't end with a new line, a
   regression introduced in the 0.12.0 release.  ([#4361][])
 
-### Enhancements and new features
-
-?
+- The clone command failed to git-annex-init a fresh clone whenever
+  it considered to add the origin of the origin as a remote.  ([#4367][])
```
Comment (kyleam) [inline CHANGELOG.md:36]: CHANGELOG looks good to me aside form the missing link.  If you put in `(#4367)`, tools/link_issues_CHANGELOG will link-ify it for you.

File: datalad/version.py [hunk 2]
```diff
@@ -15,7 +15,7 @@
 # Hard coded version, to be done by release process,
 # it is also "parsed" (not imported) by setup.py, that is why assigned as
 # __hardcoded_version__ later and not vise versa
-__version__ = '0.12.4'
+__version__ = '0.12.55555'
```
Comment (kyleam) [inline datalad/version.py:18]: Why so many 5's? :)

Comment (bpoldrack) [reply datalad/version.py:18]: Ooops.

File: CHANGELOG.md (hunk 1 above)
Comment (bpoldrack) [reply CHANGELOG.md:36]: That's still confusing to me. All the other issues are referenced the exact same way.

Comment (bpoldrack) [reply CHANGELOG.md:36]: Ah, no. Now I found it (and got it hopefully). Thx!

**Pull request closed without merging**